client/lib/plans/features-list.tsx

```diff
@@ -319,7 +322,10 @@ export type FeatureObject = {
 	getCompareSubtitle?: () => TranslateResult;
 	getIcon?: () => string | { icon: string; component: MemoExoticComponent< any > } | JSX.Element;
 	isPlan?: boolean;
+	isAddOn?: boolean;
 	getFeatureGroup?: () => string;
+	getQuantity?: () => number;
+	getUnitProductSlug?: () => string;
```

Comment:
Let's add a comment here explaining similarly.

Reply:
Resolved in [5e11208](https://github.com/Automattic/wp-calypso/pull/79041/commits/5e11208a395a7d72e2c98d79708e0ef8cd3149e6)